index.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  extends: ["stylelint-config-standard", "stylelint-config-prettier"],
```

Comment:
![memo](https://img.shields.io/badge/review-memo-white.svg)
Ref: https://stylelint.io/migration-guide/to-15#deprecated-stylistic-rules.